### DISCUSSION TITLE
blog/rusia-se-afianza-en-el-juego-geopolitico-por-latinoamerica-junto-a-venezuela-y-cuba

### DISCUSSION THREAD

**Discussion by giscus** [olivaresrafael/eko.red#4] [General]
# blog/rusia-se-afianza-en-el-juego-geopolitico-por-latinoamerica-junto-a-venezuela-y-cuba

Qué tanta importancia podría tener Venezuela y Cuba para Rusia, para que el secretario del Consejo de Seguridad, Nikolái Pátrushev, se tomara un tiempo para viajar al Caribe y reunirse con los gobernantes, Miguel Díaz Canel de Cuba y Nicolás Maduro de Venezuela, en momentos que protagoniza una invasión militar contra Ucrania que le ha dejado 180 mil muertos a su propio ejército y daños severos a su economía. Para Pátrushev, la guerra contra Ucrania es ahora un conflicto contra la OTAN y Latinoamérica podría jugar un rol clave en ese conflicto.

https://eko.red/blog/rusia-se-afianza-en-el-juego-geopolitico-por-latinoamerica-junto-a-venezuela-y-cuba

**Comment by rafaolivares** [comment]
prueba
